### PR TITLE
Eli fix npm deploy

### PR DESCRIPTION
**Pull request by elfisher**
# Fix linting and add update npm deploy

### CODE REVIEW

File: src/components/inspectorFields/FileField.js [hunk 1]
```diff
@@ -28,13 +32,15 @@ class FileField extends React.Component {
         className="form-control input-like-button"
         accept={accept}
         maxSize={maxSize}
-        onDropAccepted={e => this.handleChange(e)}>
+        onDropAccepted={e => this.handleChange(e)}
```
Comment (KiaFathi) [inline src/components/inspectorFields/FileField.js]: This is bad practice, recreates the function on every render, and react re-renders very frequently in javascript virtual dom. Bind the responder in the constructor.

https://stackoverflow.com/questions/36677733/why-jsx-props-should-not-use-arrow-functions

File: src/components/inspectorFields/NumberField.js [hunk 2]
```diff
@@ -7,6 +7,10 @@ const propTypes = {
   value: PropTypes.any,
 };
 
+const defaultProps = {
+  value: null,
+};
+
 class NumberField extends React.Component {
   handleChange(e) {
```
Comment (KiaFathi) [inline src/components/inspectorFields/NumberField.js]: See above comment.

File: src/components/inspectorFields/SelectionField.js [hunk 3]
```diff
@@ -9,6 +9,10 @@ const propTypes = {
   strings: PropTypes.object.isRequired,
 };
 
+const defaultProps = {
+  value: null,
+};
+
 class SelectionField extends React.Component {
   handleChange(e) {
```
Comment (KiaFathi) [inline src/components/inspectorFields/SelectionField.js]: See above comment.

**Comment by elfisher** [comment]
@KiaFathi removed the arrow functions and use binding as the guide you sent suggests.

**Pull request merged**